source/js/openstreetmap/helpers/osmHelpers.js

```diff
@@ -0,0 +1,64 @@
+export function zoomToMarker(marker) {
+    if (marker && marker.__parent) {
+        const cluster = marker.__parent;
+        cluster.zoomToBounds();
+        setTimeout(function () {
+            marker.openPopup();
+        }, 300);
+    }
+}
+
+export function createMarkerElementPairs(map, markers, container) {
+    if (!map || !markers || !container) return;
+    const sidebar = container.querySelector('.c-openstreetmap__sidebar');
+    const sidebarItems = sidebar.querySelectorAll('[js-map-lng][js-map-lat]');
```

Comment:
sidebar is possibly null.
```suggestion
    const sidebarItems = sidebar?.querySelectorAll('[js-map-lng][js-map-lat]') ?? [];
```